File: test4.cpp
#include<iostream>
#include<string>

int main()
{
  int n, max = 0, min = 0;

  std::cin >> n;

  std::string *name = new std::string[n];
  std::string *number = new std::string[n];
  int *score = new int[n];

  for(int i = 0; i < n; i++)
  {
    std::cin >> name[i] >> number[i] >> score[i];
  }

  for(int i = 0; i < n; i++)
  {
    if(score[i] > score[max])
      max = i;
    else if(score[i] < score[min])
      min = i;
  }

  std::cout << name[max] << " " << number[max] << std::endl;
  std::cout << name[min] << " " << number[min] << std::endl;

  system("pause");
}

File: test4.py
n = max = min = 0

n = eval(input())

name = []
number = []
score = []

for i in range(n):
  namet, numbert, scoret= input().split()
  name.append(namet)
  number.append(numbert)
  score.append(eval(scoret))

for i in range(n):
  if(score[i] > score[max]):
    max = i
  elif(score[i] < score[min]):
    min = i

print(name[max], end = " ")
print(number[max])
print(name[min], end = " ")
print(number[min])
